src/orion/core/worker/experiment.py

```diff
@@ -573,7 +573,7 @@ def configure(self, config, enable_branching=True, enable_update=True):
             raise DuplicateKeyError("Cannot register an existing experiment with a new config")
 
         # Copy and simulate instantiating given configuration
-        experiment = Experiment(self.name)
+        experiment = Experiment(self.name, version=self.version)
```

Comment:
How was this not detected by unit-tests and for which tests was it now required?

Reply:
Because there are no tests for ExperimentNode.

Reply:
We should add a unittest in `TestConfigProperty` to verify that configuring an experiment with version=1 when version 2 exists does not increment to 3.